### PR TITLE
scheduler(ticdc): remove v2 scheduler

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/4954

### What is changed and how it works?
remove v2 scheduler and its tests.

#### Tests 

 - Unit test
 - Integration test

##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/scheduler/internal/info_provider.go [hunk 1]
```diff
@@ -32,18 +32,4 @@ type InfoProvider interface {
 	// TODO: Remove the API, as capture no longer has local checkpoint ts and
```
Comment (hi-rustin) [inline cdc/scheduler/internal/info_provider.go:32]: @overvenus Do we need to remove it this time?

File: cdc/owner/changefeed.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ import (
 	"go.uber.org/zap"
 )
 
-// newSchedulerFromCtx creates a new schedulerV2 from context.
+// newSchedulerFromCtx creates a new schedulerV3 from context.
 // This function is factored out to facilitate unit testing.
```
Comment (3AceShowHand) [inline cdc/owner/changefeed.go:45]: after `schedulerV2` removed, there is only one scheduler implementation in the cluster, so no need to name it `schedulerV3`, just name it `scheduler` should be ok?

**Review by 3AceShowHand — APPROVED** [review]

File: cdc/scheduler/internal/info_provider.go (hunk 1 above)
Comment (overvenus) [reply cdc/scheduler/internal/info_provider.go:32]: Yes, it's dead code now.

File: cdc/owner/changefeed.go (hunk 2 above)
Comment (hi-rustin) [reply cdc/owner/changefeed.go:45]: yes. I'll update it. Thanks!

File: cdc/scheduler/internal/info_provider.go (hunk 1 above)
Comment (hi-rustin) [reply cdc/scheduler/internal/info_provider.go]: Removed! Thanks!

File: cdc/model/http_model.go [hunk 3]
```diff
@@ -190,16 +190,7 @@ type ProcessorCommonInfo struct {
 
 // ProcessorDetail holds the detail info of a processor
 type ProcessorDetail struct {
-	// The maximum event CommitTs that has been synchronized.
-	CheckPointTs uint64 `json:"checkpoint_ts"`
-	// The event that satisfies CommitTs <= ResolvedTs can be synchronized.
-	ResolvedTs uint64 `json:"resolved_ts"`
-	// all table ids that this processor are replicating
```
Comment (overvenus) [inline cdc/model/http_model.go:197]: Please keep the comment.

Comment (hi-rustin) [reply cdc/model/http_model.go:197]: Reverted!

**Review by asddongmen — APPROVED** [review]

**Review by hi-rustin — COMMENTED** [review]
Self check

**Pull request merged**